src/libs/ReportUtils.js

```diff
@@ -754,6 +754,17 @@ function isMoneyRequestReport(reportOrID) {
     return isIOUReport(report) || isExpenseReport(report);
 }
 
+/**
```

Comment:
Why is this a part of this PR?

Reply:
Okey, I think I get it. The added usage cause a lint error.

I'm not sure I understand it though, to be honest, I thought that there was no problem with calling functions in the same module, even when there are cycles.

Reply:
It was causing a lint error due to the function being used before it was defined so I needed to rearrange its order.